### ISSUE TITLE
Test Swedish LW Analysis Portal

### ISSUE THREAD

**Issue by hannusaarenmaa** [eubon/biodiversity-portal#1]
Test https://www.analysisportal.se/ and decide whether the functionalities are right for the EU BON portal.  It needs to be scaled to EU level, but is there anything else that is different for what EU BON needs?   Can the current prototype be changed to work like this?  What would it take?

**Comment by fcoantonio** [comment]
I agree, but we should focus on those functionalities related to metadata analysis and visualisation, given that ost of our data providers are actually "metadata providers". The registry and metadata catalogue (or GI-cat) will give as and endpoint to search and get metadata, e.g. using CSW services, providing links to the data files (these links will be part of the metadata).  
For data visualisation and time-based visualisation, the alternative would be getting all the occurrence data of GBIF and connect them to visualising components, e.g. a CartoDB server instance, as we commented in Cambridge.
